Makefile

```diff
@@ -158,24 +158,24 @@ define CROSSBUILD_HELP_INFO
 #   WHAT: Component names to be lint check. support: $(BINARIES)
 #         If not specified, "everything" will be cross build.
 #
-# GOARM: go arm value, now support:$(GOARM_VALUES)
+# ARM_VERSION: go arm value, now support:$(GOARM_VALUES)
 #        If not specified, build binary for ARMv8 by default.
 #
 #
 # Example:
 #   make crossbuild
 #   make crossbuild HELP=y
 #   make crossbuild WHAT=edgecore
-#   make crossbuild WHAT=edgecore GOARM=GOARM7
+#   make crossbuild WHAT=edgecore ARM_VERSION=GOARM7
 #
 endef
 .PHONY: crossbuild
 ifeq ($(HELP),y)
 crossbuild:
 	@echo "$$CROSSBUILD_HELP_INFO"
 else
-crossbuild: clean
```

Comment:
Remove the clean here, or if we build one binary, then build another one, the previous will be deleted.